.github/workflows/idefix-ci-doc.yml

```diff
@@ -0,0 +1,26 @@
+name: Idefix CIs
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+      - develop
+  pull_request:
+    paths-ignore:
+      - '.github/ISSUE_TEMPLATE/*'
+
+
+jobs:
+  ReadTheDocs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repo
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
```

Comment:
Is this actually needed here ?

Reply:
yes, I need the repo. The definitions from Kokkos are also needed for doxygen I believe.